eras/dijkstra/src/Cardano/Ledger/Dijkstra/PParams.hs

```diff
@@ -1,59 +1,383 @@
+  , dppPrices :: !(THKD ('PPGroups 'EconomicGroup 'NoStakePoolGroup) f Prices)
+  -- ^ Prices of execution units (for non-native script languages)
+  , dppMaxTxExUnits :: !(THKD ('PPGroups 'NetworkGroup 'NoStakePoolGroup) f OrdExUnits)
+  -- ^ Max total script execution resources units allowed per tx
+  , dppMaxBlockExUnits :: !(THKD ('PPGroups 'NetworkGroup 'SecurityGroup) f OrdExUnits)
+  -- ^ Max total script execution resources units allowed per block
+  , dppMaxValSize :: !(THKD ('PPGroups 'NetworkGroup 'SecurityGroup) f Word32)
+  -- ^ Max size of a Value in an output
+  , dppCollateralPercentage :: !(THKD ('PPGroups 'TechnicalGroup 'NoStakePoolGroup) f Word16)
+  -- ^ Percentage of the txfee which must be provided as collateral when
+  -- including non-native scripts.
+  , dppMaxCollateralInputs :: !(THKD ('PPGroups 'NetworkGroup 'NoStakePoolGroup) f Word16)
+  -- ^ Maximum number of collateral inputs allowed in a transaction
+  , -- New ones for Dijkstra:
+    dppPoolVotingThresholds :: !(THKD ('PPGroups 'GovGroup 'NoStakePoolGroup) f PoolVotingThresholds)
+  -- ^ Thresholds for SPO votes
+  , dppDRepVotingThresholds :: !(THKD ('PPGroups 'GovGroup 'NoStakePoolGroup) f DRepVotingThresholds)
+  -- ^ Thresholds for DRep votes
+  , dppCommitteeMinSize :: !(THKD ('PPGroups 'GovGroup 'NoStakePoolGroup) f Word16)
+  -- ^ Minimum size of the Constitutional Committee
+  , dppCommitteeMaxTermLength :: !(THKD ('PPGroups 'GovGroup 'NoStakePoolGroup) f EpochInterval)
+  -- ^ The Constitutional Committee Term limit in number of Slots
+  , dppGovActionLifetime :: !(THKD ('PPGroups 'GovGroup 'NoStakePoolGroup) f EpochInterval)
+  -- ^ Gov action lifetime in number of Epochs
+  , dppGovActionDeposit :: !(THKD ('PPGroups 'GovGroup 'SecurityGroup) f Coin)
+  -- ^ The amount of the Gov Action deposit
+  , dppDRepDeposit :: !(THKD ('PPGroups 'GovGroup 'NoStakePoolGroup) f Coin)
+  -- ^ The amount of a DRep registration deposit
+  , dppDRepActivity :: !(THKD ('PPGroups 'GovGroup 'NoStakePoolGroup) f EpochInterval)
+  -- ^ The number of Epochs that a DRep can perform no activity without losing their @Active@ status.
+  , dppMinFeeRefScriptCostPerByte ::
+      !(THKD ('PPGroups 'EconomicGroup 'SecurityGroup) f NonNegativeInterval)
+  -- ^ Reference scripts fee for the minimum fee calculation
+  -- TODO ensure that the groups here make sense
+  , dppMaxRefScriptSizePerBlock :: !(THKD ('PPGroups 'NetworkGroup 'SecurityGroup) f Word32)
+  -- ^ Limit on the total number of bytes of all reference scripts combined from
+  -- all transactions within a block.
+  , dppMaxRefScriptSizePerTx :: !(THKD ('PPGroups 'NetworkGroup 'SecurityGroup) f Word32)
+  -- ^ Limit on the total number of bytes of reference scripts that a transaction can use.
+  , dppRefScriptCostStride :: !(THKD ('PPGroups 'NetworkGroup 'SecurityGroup) f Word32)
+  , dppRefScriptCostMultiplier :: !(THKD ('PPGroups 'NetworkGroup 'SecurityGroup) f NonNegativeInterval)
```

Comment:
My apologies. I forgot that those two values can't be zero. That's why CI is failing
```suggestion
  , dppRefScriptCostStride :: !(THKD ('PPGroups 'NetworkGroup 'SecurityGroup) f (NonZero Word32))
  , dppRefScriptCostMultiplier :: !(THKD ('PPGroups 'NetworkGroup 'SecurityGroup) f PositiveInterval)
```

Also lenses and upgrade parameters in genesis will need their types updated too.